pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>mod-circulation</artifactId>
   <groupId>org.folio</groupId>
-  <version>16.4.0-SNAPSHOT</version>
+  <version>16.3.0-TESTING</version>
```

Comment:
TESTING?

Reply:
it was an accidental push, I removed it in the latest commit